clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -494,6 +496,42 @@ private void handleTableCommand() {
     }
   }
 
+  /** Handles the command execution for Models based on command type and the command line options. */
+  private void handleModelCommand() {
+    String url = getUrl();
+    String auth = getAuth();
+    String userName = line.getOptionValue(GravitinoOptions.LOGIN);
+    FullName name = new FullName(line);
+    String metalake = name.getMetalakeName();
+    String catalog = name.getCatalogName();
+
+    Command.setAuthenticationMode(auth, userName);
+    List<String> missingEntities = Lists.newArrayList();
+    if (metalake == null) missingEntities.add(CommandEntities.METALAKE);
+    if (catalog == null) missingEntities.add(CommandEntities.CATALOG);
+
+    // TODO: Add remaining commands to TestableCommandLine
+    String model = name.getModelName();
+    if (model == null) {
+      missingEntities.add(CommandEntities.MODEL);
```

Comment:
This is getting complicated ... unnecessarily.
Is it possible to postpone these argument checking to the specific command action handlers? For example, we only check the arguments in, say, `newModelAudit` layer.
It is hard, if possible at all, to do a unified check at this layer.
For example, you may want to list models, in which case the model name would be empty.

Reply:
There is a separate issue on adding a validate method to commands, but until that is done this is OK.

Reply:
Agreed. Let's keep the scope of this PR.